Utilities/VisualStudioProjectTemplates/EdFi.ProjectTemplates.Installer/Properties/AssemblyInfo.cs

```diff
@@ -14,7 +14,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("Ed-Fi Project Templates Installer")]
-[assembly: AssemblyCopyright("Copyright © Ed-Fi Alliance 2018")]
+[assembly: AssemblyCopyright("Copyright © Ed-Fi Alliance  + DateTime.now().year")]
```

Comment:
and Contributors is missing here .  And also check required wordings "Copyright  © $(date +'%Y') Ed-Fi Alliance, LLC and Contributors"